assert_test.go

```diff
@@ -47,3 +47,33 @@ func NotEqualf(t *testing.T, src interface{}, dst interface{}, msg string, args
 		t.Fatalf("equal %v and %v. %s", src, dst, fmt.Sprintf(msg, args...))
 	}
 }
+
+func IsNil(t *testing.T, src interface{}) {
+	IsNilf(t, src, "")
+}
+
+// reference: https://github.com/stretchr/testify/blob/v1.5.1/assert/assertions.go#L507
+func IsNilf(t *testing.T, src interface{}, msg string, args ...interface{}) {
```

Comment:
`reflection.DeepEqual` does not handle nil equality.

This implementation is based on https://github.com/stretchr/testify.
I think it's better to import stretcher/testify if such assertions are needed more.